mypy/util.py

```diff
@@ -205,6 +203,7 @@ def correct_relative_import(cur_mod_id: str,
 
 
 def get_class_descriptors(cls: 'Type[object]') -> Sequence[str]:
+    import inspect  # Lazy import for minor startup speed win
```

Comment:
What were you timing? I think `get_class_descriptors` is unfortunately actually on the path for a lot of stuff, even not the daemon, so I suspect this won't speed up real workloads? Though I'm not sure.

Reply:
I might have been timing `mypy --help` for this. I timed other changes by type checking a small program. Since this is a very small change I think it's okay to keep it even if it doesn't help in any realistic scenarios.